code/modules/reagents/reagent_containers/borghydro.dm

```diff
@@ -70,8 +70,10 @@
 			return
 
 	if (M.can_inject(user, 1))
-		user << "<span class='notice'>You inject [M] with the injector.</span>"
-		M << "<span class='notice'>You feel a tiny prick!</span>"
+		visible_message("<span class='notice'>[user] starts to inject [M] with their hypospray!</span>")
+		if(do_mob(user, 3 SECONDS, act_target = M))

```

Comment:
Wrong param order:
`/proc/do_mob(mob/user , mob/target, delay = 30, numticks = 10, needhand = TRUE, display_progress = TRUE)`

Should be: `do_mob(user, M, 3 SECONDS)`